### ISSUE TITLE
Rancher could display a warning banner when creating k3s custom cluster 

### ISSUE THREAD

**Issue by IsaSih** [rancher/dashboard#10475]
Considering that Kubernetes versions newer than 1.27 will stop supporting in-tree cloud-providers, even though k3s doesn’t have any cloud provider built-in, we could inform the user with a warning banner that configuring a cloud-provider whether through rancher or standalone is a manual/unsupported process".

Currently, when the user selects a rke2 provider, the UI displays a dropdown list of cloud providers and when the user switches to k3s, the dropdown list of cloud providers is hidden. If the user is expecting to configure a cloud provider via the UI, this warning message will help them understand why the dropdown list of cloud providers is not available.

Current behavior:

Uploading Screen Recording 2024-02-13 at 5.49.25 PM.mov…

**Describe the solution you'd like**
Rancher could have a warning message in the UI explaining that the user has to go over the process manually instead of just hiding the dropdown list of cloud providers with no other explanations.

**Comment by richard-cox** [comment]
@kwwii / @mantis-toboggan-md is this something we want to do?

**Comment by kwwii** [comment]
I don't think that we document similar feature differences in the UI unless there is a clear customer need, which I don't see in this case. I don't think that the behaviour for K3S has changed and therefor does not need an extra call-out.

**Comment by nwmac** [comment]
Closing as won't fix
